tests/integration/states/test_file.py

```diff
@@ -3866,3 +3867,54 @@ def test_file_managed_http_source_skip_verify(self):
                              skip_verify=True)
         log.debug('ret = %s', ret)
         self.assertSaltTrueReturn(ret)
+
+
+@destructiveTest
+@skipIf(not salt.utils.is_windows(), 'windows test only')
+class WinFileTest(ModuleCase):
+    '''
+    Test for the file state on Windows
+    '''
+    def setUp(self):
+        self.test = self.run_state('file.managed', name='c:/testfile', makedirs=True, contents='Only a test')
+
+    def tearDown(self):
+        del self.test
+
+    def test_file_managed(self):
+        '''
+        Test file.managed on Windows
+        '''
+        self.assertTrue(self.run_state('file.exists', name='c:/testfile'))
+
+    def test_file_copy(self):
+        '''
+        Test file.copy on Windows
+        '''
+        ret = self.run_state('file.copy', name='c:/testfile_copy', makedirs=True, source='c:/testfile')
+        self.assertTrue(ret)
+
+    def test_file_comment(self):
+        '''
+        Test file.comment on Windows
+        '''
+        self.run_state('file.comment', name='c:/testfile', regex='^Only')
+        file = 'c:/testfile'
```

Comment:
since it looks like you are using `c:/testfile` in a lot of cases here might be better to set as a class variable